wiki/Tournaments/OWC/2013/de.md

```diff
@@ -458,40 +458,42 @@ Das Turnier kam zu einem Ende mit folgenden Platzierungen:
 ### Partieanweisungen
 
 1. Ein Schiedsrichter wird 30 Minuten im Vorhinein ein Multiplayer Raum erstellen. Die Spieler müssen sich während diesem Zeitraum versammeln.
-   1. Der Raum wird verschlossen. Das Passwort und die Einladung zum Raum werden beiden Kaptiänen so früh wie möglich gesendet.
-   2. Raumeinstellungen sind osu!, Team-Vs., Win Condition: 'Score'. Raumname muss "osu! World Cup 2013: TeamBlue vs TeamRed" lauten.
-   3. Das Team, welches zuerst im Raumnamen genannt wurde, wird das blaue Team, das andere Team dementsprechend dann das rote Team.
+   - Der Raum wird verschlossen. Das Passwort und die Einladung zum Raum werden beiden Kaptiänen so früh wie möglich gesendet.
+   - Raumeinstellungen sind osu!, Team-Vs., Win Condition: 'Score'. Raumname muss "osu! World Cup 2013: TeamBlue vs TeamRed" lauten.
+   - Das Team, welches zuerst im Raumnamen genannt wurde, wird das blaue Team, das andere Team dementsprechend dann das rote Team.
 2. Der Schiedsrichter wird dem Raum extern beitreten und mit Hilfe des Multi-Spectator Clients die Partie verfolgen.
 3. Den Spielern steht es frei zu eine Beatmap für die "Warm-Up" Phase zu spielen.
-   1. Die Kapitäne können frei nach Belieben eine Beatmap aus dem NoMod Stappel auswählen.
-   2. Die Auswahl aus dem mod-spezifischen Stappeln ist limitiert. Jeder Kapitän darf eine mod-spezifische Beatmap aus dem jeweiligen Stappel während der Partie auswählen.
-      1. In der K.-o.-Runde kann auf den FreeMod Stappel frei zugegriffen werden.
-   3. Im Falle eines Unentschiedens, muss die Tiebreaker Map gespielt werden.
-   4. Die Kapitäne müssen dem Schiedsrichter mitteilen, welche Map gewählt wurde.
-4. Jeder Kapitän muss die Ergebnisse mit einem Screenshot abspeichern. Die Schiedsrichter werden den Kapitänen daran erinnern.
-5. Die Ergebnisse werden per Statistiken auf einer Webseite veröffentlicht.
+4. Die Mapauswahl erfolgt abwechselnd durch jeden Kapitän, der eine Map aus dem Mappool auswählt. Jeder Kapitän muss einmal `!roll` in `#multiplayer` verwenden, um zu bestimmen, welches Team zuerst wählt.
+   - Die Kapitäne können frei nach Belieben eine Beatmap aus dem NoMod Sapel auswählen.
```

Comment:
```suggestion
   - Die Kapitäne können frei nach Belieben eine Beatmap aus dem NoMod Stapel auswählen.
```
Okay, now I'm curious. Is Sapel a word? 👀

Reply:
Nope, i just missed these. There were other typos like this written as `Stappel` so i replaced all of those with `Stapel` instead, but missed other typos like Sapel. lol